vms/vms/static/vms/js/timepicker.js

```diff
@@ -1,12 +1,13 @@
 $(function() {
-    $('#datetimepicker1').datetimepicker({
-      pickDate: false,
-	  pickSeconds: false
-    });
+  $('#datetimepicker1').datetimepicker({
+    pickDate: false,
+    pickSeconds: false
+  });
+});
+
+$(function() {
+  $('#datetimepicker2').datetimepicker({

```

Comment:
if both of them are doing the same, then probably what we might need is a class and using it once. But since this is a different improvement - maybe another time.

Reply:
@tonythomas01 We can create a new issue for this.